requirements.txt

```diff
@@ -0,0 +1 @@
+pyyaml
```

Comment:
By the way, why are you using this instead of [install_requires]?

Reply:
To elaborate, [setup_requires] and [test_requires] are deprecated because they use easy_install, [install_requires] is not.

Reply:
Could you point me to where this is described? In the `setuptools` docs the `tests_require` is said to use pip if available https://setuptools.readthedocs.io/en/latest/setuptools.html. `easy_install` is seriously deprecated, I was not aware that it was being used in the case that `setup_requires` or `test_requires` was specified.

That being said, I'll place the content of `requirements.txt` in `install_requires` instead.

(We could wait a little longer probably, but the entire `setup.py` is deprecated, and instead we should move towards `pyproject.toml`)

Reply:
https://github.com/pypa/setuptools/issues/1684#issuecomment-508156856 does mention easy_install when using `setup.py test`, but that means the docs are incorrect?

Reply:
> Setuptools offers the setup_requires setup() keyword for specifying dependencies that need to be present in order for the setup.py script to run. Internally, Setuptools uses easy_install to fulfill these dependencies.

https://pip.readthedocs.io/en/1.4.1/cookbook.html#controlling-setup-requires

Reply:
Yeh.. Having to refer to `pip`s documentation in order to read about `setuptools`'s behaviour is a bit annoying.. But coming to think about it, I believe I've seen the `easy_install` command being used when running tests..